Eduplatform_site/account/models.py

```diff
@@ -39,29 +42,41 @@ class Meta:
         verbose_name_plural = _("teachers")
 
 
-class Student(models.Model):
+class Student(models.Model,DateTimeMixinModel):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
+    course = models.ForeignKey(learning_models.Course, on_delete= models.CASCADE,null=True)
+    group = models.ForeignKey('Group',on_delete=models.CASCADE,null=True)
+    completed_tests = models.JSONField(default=dict)
```

Comment:
rating - floatfield
completed_tests - M2M(Test)
complited_articles - M2M(Article)

Reply:
Students union in group, not group in students,
course also delete it should be in group model